rts/motoko-rts/src/region.rs

```diff
@@ -6,8 +6,8 @@ use crate::types::{size_of, Blob, Bytes, Region, Value, TAG_REGION};
 // Versions
 // Should agree with constants in StableMem in compile.ml
 // Version 0 to 2 are legacy.
-const VERSION_STABLE_HEAP_NO_REGIONS: u32 = 3;
-const VERSION_STABLE_HEAP_REGIONS: u32 = 4;
+const VERSION_STABLE_HEAP_NO_REGIONS: usize = 3;
+const VERSION_STABLE_HEAP_REGIONS: usize = 4;
```

Comment:
I guess I would have kept these as u32s (here and in compile.ml) since that is the size of the value written to the stable memory metadata, but I guess it fine to widen internally. The size in meta-data is unchanged I think and remains 32-bit (so we can read the metadata of a 32-bit canister when upgrading).